lib/elixir/lib/calendar/date.ex

```diff
@@ -398,6 +398,43 @@ defmodule Date do
     end
   end
 
+  @doc """
+  Converts a number of gregorian days to a `Date` struct.
+
+  ## Examples
+
+      iex> Date.from_gregorian_days(1)
+      ~D[0000-01-02]
+      iex> Date.from_gregorian_days(730_485)
+      ~D[2000-01-01]
+      iex> Date.from_gregorian_days(-1)
+      ~D[-0001-12-31]
+
+  """
+  @spec from_gregorian_days(integer(), Calendar.calendar()) :: t
```

Comment:
I think we could document the version in which these new functions are introduced 🙂 

```suggestion
  @doc since: "1.11.0"
  @spec from_gregorian_days(integer(), Calendar.calendar()) :: t
```

The same applies for the rest of the functions.

Reply:
Done